addrmgr/network.go

```diff
@@ -115,24 +102,15 @@ func isLocal(netIP net.IP) bool {
 	return netIP.IsLoopback() || zero4Net.Contains(netIP)
 }
 
-// isOnionCatTor returns whether or not the passed address is in the IPv6 range
-// used by bitcoin to support Tor (fd87:d87e:eb43::/48).  Note that this range
-// is the same range used by OnionCat, which is part of the RFC4193 unique local
-// IPv6 range.
-func isOnionCatTor(netIP net.IP) bool {
-	return onionCatNet.Contains(netIP)
-}
-
 // NetAddressType is used to indicate which network a network address belongs
 // to.
 type NetAddressType uint8
 
 const (
-	UnknownAddressType NetAddressType = iota
-	IPv4Address
-	IPv6Address
-	TORv2Address
-	TORv3Address
+	UnknownAddressType NetAddressType = 0
```

Comment:
Should call out why it's not using `iota`, imo (because it's used in the serialization, etc).  There are examples in various parts of the code base.